src/pages/Version/is-package-version-valid.ts

```diff
@@ -0,0 +1,16 @@
+import { PackageMetaInterface } from '../../../types/packageMeta';
+
+function isPackageVersionValid(packageMeta: Partial<PackageMetaInterface>, packageVersion?: string): boolean {
+  if (!packageVersion || typeof packageVersion === 'undefined') {
```

Comment:
This is also a utilities. The main reason is is much easier to `jest.mock` one file rather 10 in the same unit test.